assets/components/pages/project/curate/CurateVariantPage.js

```diff
@@ -292,7 +296,7 @@ class CurateVariantPage extends React.Component {
                           <ResultVerdict />
                         </span>
                         <span style={{ marginRight: "10px" }}>
-                          <VerdictValidIcon result={result} />
+                          <VerdictValid />
```

Comment:
What is VerdictValid?

Reply:
It's a connected redux component using the same pattern as `<ResultVerdict />` - a wrapper to ensure that the icon updates in real-time whenever flags are changed, rather than waiting for a save & refresh.